libraries/pbrlib/genglsl/lib/mx_bsdfs.glsl

```diff
@@ -56,12 +78,48 @@ float mx_microfacet_ggx_smith_G(float NdotL, float NdotV, float alpha)
     return mx_microfacet_ggx_G1(NdotL, alpha) * mx_microfacet_ggx_G1(NdotV, alpha);
 }
 
+// http://blog.selfshadow.com/publications/s2017-shading-course/imageworks/s2017_pbs_imageworks_sheen.pdf (Equation 2)
+float mx_microfacet_sheen_NDF(float cosTheta, float roughness)
+{
+    float invRoughness = 1.0 / roughness;
```

Comment:
Need a divide by 0 check here.

Reply:
The roughness is assumed to never be zero here. A clamp to (M_FLOAT_EPS, 1.0) is done before this call is made in mx_sheen_brdf_reflection, because the clamped value is used elsewhere after the call and we avoid doing it twice.

But I can add a code comment about that here for clarity.